t/lib/ext-plugin.lua

```diff
@@ -373,6 +373,40 @@ function _M.go(case)
                 {"X-Req", "bar"},
                 {"Content-Type", "application/json"},
                 {"Content-Encoding", "deflate"},
+                {"X-Same", "one"},
```

Comment:
We don't need X-Same for other header tests?

Reply:
Sorry, I optimized the code.I divided it into three test cases based on this feature.
TEST 19: rewrite response header and call the upstream service
TEST 20: rewrite non-important response headers and call the upstream service
TEST 25: rewrite same response headers and call the upstream service

Reply:
Don't worry, it's just a small problem.